hbase-server/src/test/java/org/apache/hadoop/hbase/replication/TestReplicationSource.java

```diff
@@ -208,7 +208,8 @@ protected void doStop() {
         // completes
       }
     };
-    replicationEndpoint.start();
+    replicationEndpoint.startAsync();
```

Comment:
Our ReplicationEndpoint interface extends Guava's Service interface. The Service interface has evolved. start() is replaced by startAsync() and awaitRunning()